tests/conftest.py

```diff
@@ -62,6 +63,15 @@ def create_db():
                 return True
 
 
+def load_extensions():
+    conn = connect()
+    cur = conn.cursor()
+    try:
+        cur.execute("CREATE EXTENSION vector")
+    except psycopg2.errors.DuplicateObject:
```

Comment:
With this change, it will just skip `vector` test if the extension is not available
```suggestion
    except:
```